containers/QuestionPanelContainer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import {
+  getUserActiveQuestionIdForQueue,
+  isUserActiveStaffForQueue,
+} from '../selectors'
+
+import QuestionPanel from '../components/QuestionPanel'
+
+const QuestionPanelContainer = ({
```

Comment:
This bit actually isn't necessary - `connect` is a higher-order component (yay 421 concepts!) that will take care of passing props to the wrapped component. Take a look at `NewQuestionContainer` for an example.